### PR TITLE
React Example App

### PR DESCRIPTION
**Pull request by wood198**
Everything is working and restructured!

**To properly set up the app:**

In src/Components/Auth.js replace the API_KEY and API_SECRET variable values with your application's credentials.

**OR**

If you would like to create a .env with your application credentials: create the .env in the react_example folder outside of src. Name your variables REACT_APP_API_KEY and REACT_APP_SECRET_KEY and your .env should populate the keys within the example app for you.

**To Run:**
cd into src

Use the following commands:

- npm install

- npm start

### CODE REVIEW

**Comment by BreaGaudioso** [comment]
@cleio Verified that the example app is still running and nothing changed in actual SDK code. Our team will own review for react app to save you cycles- but feel free to leave feedback too if you'd like.

File: examples/react_example/.gitignore [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+.env
```
Comment (easterjd) [inline examples/react_example/.gitignore]: You should be able to include package-lock.json here too

File: examples/react_example/src/Css/Player.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+.select p {
+  margin: 0;
+  padding: 5px 10px;
+  font-size: 15px;
+}
+
+.demobox {
+  background-color: black;
+  position: relative;
+  align: center;
```
Comment (easterjd) [inline examples/react_example/src/Css/Player.css]: Not sure if 'align' is a new property, but I can't find anything about it. Did you mean 'text-align'?

**Comment by easterjd** [comment]
Looks really great; not much to comment on here. Only thing I saw was the root level /example and /reactexample folders. They both contain node_modules files but it looks like you have a node_modules file already. Was that purposeful?

File: examples/react_example/public/index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
```
Comment (Filtos) [inline examples/react_example/public/index.html]: If you want you can change the tab icon from the default react one to whatever you want. Just include the picture in your public folder and replace favicon.ico with the picture.

File: examples/react_example/src/Css/Genre.css [hunk 4]
```diff
@@ -0,0 +1,35 @@
+.genre-btn {
+  background-color: transparent;
+  cursor: default;
```
Comment (Filtos) [inline examples/react_example/src/Css/Genre.css]: Could be a good idea to change from default to pointer. Makes it more intuitive and lets the users know that all of the genres are clickable.

**Comment by BreaGaudioso** [comment]
Nice work Ashley, merging!

**Pull request merged**